src/Window/Context.cs

```diff
@@ -37,10 +37,19 @@ public Context()
         /// Activate or deactivate the context
         /// </summary>
         /// <param name="active">True to activate, false to deactivate</param>
+        /// <returns>true on success, false on failure</returns>
         ////////////////////////////////////////////////////////////
-        public void SetActive(bool active)
+        public bool SetActive(bool active)
         {
-            sfContext_setActive(myThis, active);
+            return sfContext_setActive(myThis, active);
+        }
+
+        /// <summary>
+        /// Get the settings of the context.
+        /// </summary>
+        public ContextSettings Settings
```

Comment:
Documentation blocks should be surrounded by ////////////////////////////////////////////////////////////